Dockerfile.amd64-alpine

```diff
@@ -0,0 +1,42 @@
+FROM resin/amd64-alpine-node:0.10.44-slim
+
+# Supervisor apt dependencies
+RUN apk add --update \
+		btrfs-progs \
+		ca-certificates \
+		rsync \
+		supervisor \
+	&& rm -rf /var/cache/apk/*
+
+# Copy supervisord configuration files
+COPY config/supervisor/ /etc/supervisor/
+
+# Install dependencies
+WORKDIR /app
+COPY package.json postinstall.sh /app/
+RUN set -x \
+	&& buildDeps='g++ sqlite-dev sqlite-libs make' \
+	&& apk add --update $buildDeps \
+	&& JOBS=MAX npm install --unsafe-perm --production --no-optional \
+	&& cd /app/node_modules/sqlite3 && ./node_modules/.bin/node-pre-gyp install --build-from-source --sqlite=/usr/lib \
```

Comment:
Is it possible to pass `--build-from-source` to `npm install`? If we can it'd be nice to do that to avoid any other incompatibilities that might happen as well